### DISCUSSION TITLE
I got a problem about CSS

### DISCUSSION THREAD

**Discussion by CHEEZcaake** [community/community#61298] [Programming Help]
### Body

My css file doesn't work with html. In my computer It works so well, Without any problems. I've uploaded my finished work to GitHub however my project doesn't work with css on pages. I've checked out some articles on GitHub and Google also asked for ChatGPT but nothing helped me. I've tried adding/deleting "/" adding/deleting "/css/..." opening a new folder putting my css into it and updating my link but no.

Here's my link,
https://github.com/CHEEZcaake/BillieEilish
Pages,
https://cheezcaake.github.io/BillieEilish/

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by radar-blip** [comment]
In your index.html,

Try changing this:
`<link rel="stylesheet" href="/styles.css">`

to this:
`<link rel="stylesheet" href="styles.css">`

**Reply by CHEEZcaake** [reply]
i did nothing is changing

**Comment by melvincwng** [comment]
@CHEEZcaake, the reason why your CSS isn't working with the HTML page is because the browser cannot find or reference your CSS file.

Hence, in this particular line of code over [here](https://github.com/CHEEZcaake/BillieEilish/blob/main/index.html#L7), try changing it to:

`<link rel="stylesheet" href="styles.css">` or `<link rel="stylesheet" href="./styles.css">`

Either approach should work.

Hope this resolves your question, take care 😄

**Comment by Sathyaraj-dev** [comment]
Hi,

Use this for all the HTML files, I noticed that some HTML files are using without './styles.css'.

So use this for all the HTML files: <link rel="stylesheet" href="./styles.css">

**Comment by Kberrio** [comment]
Hi @CHEEZcaake 👋🏼, you're very close to getting this issue resolved.

When you have a file like `styles.css` at the same level as another file, like `index.html`, they're like siblings in the same folder. So, if you're working in index.html and want to use styles.css, the browser already knows they're in the same place. You don't need to say which folder they're in because the browser automatically checks the current folder for the file you're asking for – it's like when you're in the same room as someone, you don't need to tell them where you are.

The issue you're encountering with your HTML files not properly utilizing the `styles.css` file appears to be due to a directory pathing problem. Let me try to explain as clearly as possible:

When you're working with files in different folders, you might need to use something called `../` to move up one level in the folder structure. It's like going back one step in a path.

In your repository, you have a _main_ folder named "BillieEilish" containing three additional folders: "Fonts," "html files," and "photo." There are also two standalone files: "index.html" and "styles.css."

Within these folders, you have various types of files. For instance, the "html files" folder holds several HTML files.

If you're working in the `index.html` file of "BillieEilish" and you want to use styles.css, you would normally do something like this:
```html
<link rel="stylesheet" type="text/css" href="styles.css">
```
This works because both files are in the same folder, so you don't need to say the folder's name.

However, consider your scenario where you're working on an `albums.html` file within the "html files" folder and wish to use `styles.css` from the "BillieEilish" folder. Given that `styles.css` is located in a different folder, you need to move up to the "BillieEilish" folder before accessing styles.css. This is where `../` comes into play – it signifies "go up one level."

So, in each .html file within the "html files" folder, you would include:
```html
<link rel="stylesheet" type="text/css" href="../styles.css">
```
The `..` effectively means "move up one level," followed by the path to the desired file.
I personally tested this approach and successfully resolved the issue myself.

Give it a try, and let me know if it works for you. 
Good luck! 👍🏼

**Comment by ighazal** [comment]
Hi I see your codes
please test
`<link rel="stylesheet" href="styles.css">`
Instead
`<link rel="stylesheet" href="./styles.css">`
I hope my answer will help you

**Comment by mehulrajdev** [comment]
Use this
<link rel="stylesheet" href="./styles.css">
